### ISSUE TITLE
Orvibo AllOne Pro

### ISSUE THREAD

**Issue by mrGottt** [sandysound/orvibo-b25-server#3]
Hello There,
I bought a AllOne Pro, wich has the b25 Socket, but I cannot set it up to work with the b25 server.
Due to this reason:
I also use dnsmasq to redirect  homemate.orvibo.com to my raspi that runs the b25-server.
When running nslookup I get my raspis ip.
When typing "http://homemate.orvibo.com:10001/" into the browser the server reacts (gets an error but that is fine).
But when I do whireshark Package sniffing, the AllOne still sends its Packages to 52.57.212.91.
My reverse DNS lookup found this URL for the IP:
ec2-52-57-212-91.eu-central-1.compute.amazonaws.com

I also redirected this th my raspi, but still no results.

Anyone had a similar problem?

**Comment by mrGottt** [comment]
Update:
I added a static IP route to my raspi for the ip 52.57.212.91 and 52.28.176.170.
There I added loopbacks, using ifconfig:
ifconfig lo:1 inet 52.57.212.91 netmask 255.255.255.0 up
ifconfig lo:2 inet 52.28.176.170 netmask 255.255.255.0 up

Now the Server responds. I also retrieved the UID, which you can not inspect in the HomeMate App. (The AllOne Pro is not compatible with the WiWo app)

Feel free to close the Issue

**Comment by sandysound** [comment]
Ok, cool. Glad you got it working

**Comment by wichopy** [comment]
@mrGottt I am trying to troubleshoot my server with the same method you did. When I type "http://homemate.orvibo.com:10001/" into my browser I also get an error on my local server. 

However, when I try wiresharking the packets, I cannot see any of the packets from my B25.
I tried your ifconfig commands and it is still not working. 

I am very new to networks. Any idea why I would not be able to see the packets from my B25 in wireshark? I turned off promiscuous mode and I can see packets from other devices on my network like chromecast and phones.
